lib/rjgit.rb

```diff
@@ -163,6 +163,12 @@ def self.diff(repository, options = {})
       RJGit.convert_diff_entries(diff_entries)
     end
 
+    def self.describe(repository, ref, options = {})
```

Comment:
JGit's `DescribeCommand` also supports `match`. Perhaps you could enable this option as well?

Reply:
Done: 7cfe77d0dda6f6a63df28543006509306f60c4a4